### ISSUE TITLE
libgcc-s1 not exist in Debian

### ISSUE THREAD

**Issue by HiFiPhile** [arkane-systems/genie#76]
Hi,

I'm trying to install `genie` in Debian buster but `libgcc-s1` can't be found.

This issue has been mentioned in #74 

```
 sudo apt install -y systemd-genie
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 systemd-genie : Depends: libgcc-s1 (>= 3.0) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

sources.list : 
```
deb http://deb.debian.org/debian buster  main
deb http://deb.debian.org/debian buster-updates main
deb http://security.debian.org/debian-security/ buster/updates main
deb http://ftp.debian.org/debian buster-backports main
```

**Comment by cerebrate** [comment]
Thanks for the report; looks like I missed excluding that in the build process for _buster_ . Can you try doing a manual (i.e. `dpkg -i`) install of the attached package:

[systemd-genie_1.27_amd64.deb.gz](https://github.com/arkane-systems/genie/files/5164053/systemd-genie_1.27_amd64.deb.gz)

and let me know if that works? If so, I'll update the published one.

**Comment by HiFiPhile** [comment]
I've successfully installed this package, so far so good.

**Comment by cerebrate** [comment]
Excellent. I'll include the fix in 1.28, which should ship tomorrow.
